core/foundation/inc/ROOT/RIntegerSequence.hxx

```diff
@@ -0,0 +1,106 @@
+/// \file ROOT/RIntegerSequence.hxx
+/// \ingroup Base StdExt
+/// \author Enrico Guiraud <enrico.guiraud@cern.ch>
+/// \date 2018-04-06
+
+/*************************************************************************
+ * Copyright (C) 1995-2000, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_RIntegerSequence
+#define ROOT_RIntegerSequence
+
+#if __cplusplus > 201103L
+// no need to backport
+
+#include <utility>
+
+#else
+// backport libstdc++'s implementation (gcc7.3.1)
```

Comment:
Please add link of where you got it from if possible (and if from git either a commit number here or in our commit message) .. i.e. make it as easy as possible to find potential updates :)

Reply:
I can't be more precise than "libstdc++'s implementation shipped with gcc 7.3.1", but probably I can dig up a link to the `utility` header for this version of gcc on github.

The other backports would also benefit from such links probably :sweat_smile:

Reply:
link added in the line below